atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -171,7 +233,10 @@ class OrchestrateBuildPlugin(BuildStepPlugin):
 
     def __init__(self, tasker, workflow, platforms, build_kwargs,
                  osbs_client_config=None, worker_build_image=None,
-                 config_kwargs=None):
+                 config_kwargs=None,
+                 find_cluster_retry_delay=FIND_CLUSTER_RETRY_DELAY,
+                 failure_retry_delay=FAILURE_RETRY_DELAY,
+                 max_cluster_fails=MAX_CLUSTER_FAILS):
```

Comment:
Let's include descriptions of these two in the docstring below.

Reply:
done